reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1298 @@ title: Style
 officialDoc: https://facebook.github.io/react-native/docs/style
 ---
 
-@todo
+Styles accepted by most components (thanks to a `style` prop) are defined using
+style functions that prepare required object for React Native. The style names
+and values usually match how CSS works on the web, except names are written
+using camel casing, e.g `backgroundColor` rather than `background-color`.
+
+The `style` prop accept what the style functions return. That's the simplest way
+to pass styles. You can also pass an array or list of styles - the last style in
+the array has precedence, so you can use this to mix & inherit styles.
+
+⚠️ _Note that when a component grows in complexity, it is often cleaner to use
+[`StyleSheet.create`](/bs-react-native/en/docs/apis/StyleSheet/) to define
+several styles in one place_.
+
+We have made different styles function because React Native have various
+components that accept different styles properties. For example `View` doesn't
+accept `color` (only `Text` does).
+
+## Table of Content
+
+- [Style Example](#style-example)
+- [Style Units](#style-units)
+- [Style Function](#style-functions)
+  - [`Style.style`](#style-style)
+  - [`Style.viewStyle`](#style-viewStyle)
+  - [`Style.textStyle`](#style-textStyle)
+  - [`Style.imageStyle`](#style-imageStyle)
+- [Style Props](#style-props)
+  - [Layout Style Props](#layout-style-props)
+  - [Shadow Style Props](#shadow-style-props)
+  - [Transform Style Props](#transform-style-props)
+  - [View Style Props](#view-style-props)
+  - [Text Style Props](#text-style-props)
+  - [Image Style Props](#image-style-props)
+  - [Unsafe Style Props](#unsafe-style-props)
+- [Style Helpers](#style-helpers)
+  - [`Style.array`](#style-array`)
+  - [`Style.arrayOption`](#style-arrayoption`)
+  - [`Style.list`](#style-list`)
+  - [`Style.listOption`](#style-listoption`)
+
+## Style Example
+
+Since an example worth a thousand words...
```

Comment:
adding _is_:
Since an example is worth a thousand words...

Reply:
Fixed